app/upgrades/v19/upgrades.go

```diff
@@ -65,3 +76,28 @@ func MigrateRateLimitModule(ctx sdk.Context, k ratelimitkeeper.Keeper) {
 	hourEpoch.EpochStartHeight = ctx.BlockHeight()
 	k.SetHourEpoch(ctx, hourEpoch)
 }
+
+// Add a 10% rate limit for stTIA to Celestia and Osmosis
+func AddStTiaRateLimit(ctx sdk.Context, k ratelimitkeeper.Keeper) error {
+	addRateLimitMsgTemplate := ratelimittypes.MsgAddRateLimit{
+		Denom:          StTiaDenom,
+		MaxPercentSend: sdk.NewInt(10),
+		MaxPercentRecv: sdk.NewInt(10),
+		DurationHours:  RateLimitDurationHours,
+	}
+
+	addCelestiaMsg := addRateLimitMsgTemplate
+	addCelestiaMsg.ChannelId = CelestiaTransferChannelId
+
+	addOsmosisMsg := addRateLimitMsgTemplate
+	addOsmosisMsg.ChannelId = OsmosisTransferChannelId
+
```

Comment:
Now that we have incentivized stTia on so many different chains should we add similar messages for all the chains we are incentivizing?  Seems like right now people can move a lot of volume through other chains and we wouldn't notice.  Even if we give those other chains looser different bounds because the capital might only be from a handful of people at the moment seems like a good check

Reply:
We’re planning to switch to a global rate limit soon that should cover all these cases.

in the short term, we could probably also add neutron since that pool has a lot of liquidity but I think we can wait on the others